js-http/fairos-http-client.js

```diff
@@ -0,0 +1,623 @@
+        credentials: "include"
+    });
+}
+
+docLs = function() {
+    var data = {
+        "pod_name": podName,
+        "table_name": table
+    };
+    return fetch(host + "/doc/ls" + '?' + new URLSearchParams(data), {
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json"
+        },
+        credentials: "include"
+    });
+}
+
+docOpen = function() {
+    var data = {
+        "pod_name": podName,
+        "table_name": table
+    };
+    return fetch(host + "/doc/open", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json"
+        },
+        body: JSON.stringify(data),
+        credentials: "include"
+    });
+}
+
+docEntryPut = function() {
+    var data = {
+        "pod_name": podName,
+        "table_name": table,
+        "doc":  `{"id":"1", "first_name": "Hello1", "age": 11}`,
+    };
+    return fetch(host + "/doc/entry/put", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json"
+        },
+        body: JSON.stringify(data),
+        credentials: "include"
+    });
+}
+
+docEntryGet = function() {
+    var data = {
+        "pod_name": podName,
+        "table_name": table,
+        "id":  "1",
+    };
+    return fetch(host + "/doc/entry/get" + '?' + new URLSearchParams(data), {
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json"
+        },
+        credentials: "include"
+    });
+}
+
+docFind = function() {
+    var data = {
+        "pod_name": podName,
+        "table_name": table,
+        "expr":  `age>10`,
+    };
+    return fetch(host + "/doc/find" + '?' + new URLSearchParams(data), {
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json"
+        },
+        credentials: "include"
+    });
+}
+
+docCount = function() {
+    var data = {
+        "pod_name": podName,
+        "table_name": table,
+    };
+    return fetch(host + "/doc/count", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json"
+        },
+        body: JSON.stringify(data),
+        credentials: "include"
+    });
+}
+
+docEntryDel = function() {
+    var data = {
+        "pod_name": podName,
+        "table_name": table,
+        "id":  "1",
+    };
+    return fetch(host + "/doc/entry/del", {
+        method: "DELETE",
+        headers: {
+          "Content-Type": "application/json"
+        },
+        body: JSON.stringify(data),
+        credentials: "include"
+    });
+}
+
+docDel = function() {
+    var data = {
+        "pod_name": podName,
+        "table_name": table,
+    };
+    return fetch(host + "/doc/delete", {
+        method: "DELETE",
+        headers: {
+          "Content-Type": "application/json"
+        },
+        body: JSON.stringify(data),
+        credentials: "include"
+    });
+}
+
+apiTest();
+
+function handleResponse(response) {
+    console.log(response);
```

Comment:
After login cookie is not being set and sent with the following requests. thus seeing the following responses.
![image](https://user-images.githubusercontent.com/15252513/150901515-ba5e79a2-ce29-4d93-a5ab-e159d486f3dc.png)

Reply:
This is related to setting cookieDomain to "localhost" from server side.